build.gradle

```diff
@@ -25,7 +25,7 @@ ext {
     automatticAboutVersion = '1.4.0'
     automatticRestVersion = '1.0.8'
     automatticTracksVersion = '5.1.0'
-    gutenbergMobileVersion = 'v1.121.0-alpha2'
+    gutenbergMobileVersion = '6992-2ec1f526f28da566370a7b705176f6e9c0054c30'
```

Comment:
Friendly reminder: update this to the soon-to-be-published release tag before merging.